kategory-docs/docs/docs/README.md

```diff
@@ -4,4 +4,107 @@ title: Quick Start
 permalink: /docs/
```

Comment:
Is there a way we do not have to "copypaste" the whole Readme in the docs? Some kind of `include` ?

Reply:
not that I'm aware off since the README is outside of `jekyll` when processing the docs folder.